source/parameters/meta.md

```diff
@@ -2,17 +2,32 @@
 (s:meta.params)=
 ## MetaP: Metadata Parameters
 
-The `MetaP` parameter group can be used for metadata that describes the lattice element
-but is not part of the PALS standard. Such data is necessarily program dependent.
-Example metadata could be blueprint information, information on power supply connections, etc.
-Example:
+`MetaP` has four standard components
 ```{code} yaml
-MetaP:
-  alias:  an_alternative_name
-  Meta: Mark 4 quadrupole
+  alias         - String
+  ID            - String
+  label         - String
+  description   - String
```

Comment:
I really liked if this was a concrete example as on the left, this new syntax to describe keys is hard to read for us implementing this.

Reply:
I don't know what you mean. Please suggest something.

Reply:
Yes, let's discuss in `FloorShiftP` where I added the concrete example. We can also chat in our meeting in an hr to clarify more.